frontend/src/metabase/home/components/Onboarding/Onboarding.module.css

```diff
@@ -8,90 +8,6 @@
   }
 }
 
-.item:not([data-css-specificity-hack="🪗"]) {
```

Comment:
Extracted this to `OnboardingAccordion.module.css` so we can share it with both onboarding accordions.